examples/resources/ovirt_vm_instance_type_id/provider.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
```

Comment:
`ovirt_vm_instance_type_id` is not a separate resource and should not be added to the examples. This folder is used for doc generation.

Reply:
removed